### PR TITLE
chore(core): nx plugin submission @rxap/plugin-localazy

### PR DESCRIPTION
**Pull request by mmuenker**
Add the nx plugin [@rxap/plugin-localazy](https://www.npmjs.com/package/@rxap/plugin-localazy) to the list of approved plugins

### CODE REVIEW

File: community/approved-plugins.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 [
+  {
+    "name": "@rxap/plugin-localazy",
+    "description": "A Nx plugin for localazy.com upload and download tasks.",
```
Comment (vsavkin) [inline community/approved-plugins.json:4]: A Nx => An Nx

**Review by vsavkin — CHANGES_REQUESTED** [review]
Thank you for the PR. 

Could you fix the description?

**Comment by mmuenker** [comment]
@vsavkin the description is fixed

**Comment by vsavkin** [comment]
Thank you!

**Pull request merged**